### PR TITLE
More i18n omissions 

### PR DESCRIPTION
**Pull request by j-dornbusch**
Extracts  strings from template to allow i18n.

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by j-dornbusch** [comment]
@jmfrenzel  I think you'll want to improve German translations before the branch is merged.  
@samvera/hyrax-code-reviewers  I think the last test failure (`Selenium::WebDriver::Error::WebDriverError`) is not related to the changes I made.

**Comment by cjcolvar** [comment]
@j-dornbusch Those errors just came up in a different PR so I'm investigating.  Hopefully it can be cleared up soon!

File: config/locales/hyrax.de.yml [hunk 1]
```diff
@@ -536,6 +536,10 @@ de:
         show_more_parent_collections: Zeig mehr...
         subcollection_count: Untergeordnete Sammlungen
         works_in_collection: Arbeiten in dieser Sammlung
+      sort_and_per_page:
+        number_of_results_to_display_per_page: Anzahl der Ergebnisse pro Seite angezeigt werden
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Anzahl der Ergebnisse die pro Seite angezeigt werden

File: config/locales/hyrax.de.yml [hunk 2]
```diff
@@ -598,15 +602,22 @@ de:
             relationships: Beziehungen
             sharing: Freigaben
         form_branding:
+          alt_text: 'Alt-Text:'
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: 'Alternativer Linktext:'

File: config/locales/hyrax.de.yml [hunk 3]
```diff
@@ -598,15 +602,22 @@ de:
...
           banner:
             description: Ein Bild, das am oberen Rand der Sammlungsseite angezeigt werden soll. Für beste Ergebnisse laden Sie ein Bild (JPG, GIF oder PNG) hoch, das mindestens 120 Pixel groß und 1200 Pixel breit ist.
             label: Banner
           branding:
             description: Optional können Sie ein Bannerbild und / oder Logo-Bilder hochladen, die mit dieser Sammlung verknüpft werden sollen. Wenn diese hochgeladen werden, werden diese Bilder oben auf der Sammlungsseite angezeigt, um ein eindeutiges Markenbildung für die Sammlung bereitzustellen.
             label: Markenbildung
+          choose_file: Wählen Sie "Datei"
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: "Datei" wählen

File: config/locales/hyrax.de.yml [hunk 4]
```diff
@@ -598,15 +602,22 @@ de:
...
+          link_url: 'Link-URL:'
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: 'URL-Link:'

File: config/locales/hyrax.de.yml [hunk 5]
```diff
@@ -598,15 +602,22 @@ de:
...
           logo:
             description: Ein oder mehrere Bilder, die oben auf der Sammlungsseite angezeigt werden. Für beste Ergebnisse laden Sie ein Bild (JPG, GIF oder PNG) hoch, das 40 Pixel hoch ist. Größere Bilder werden auf 40 Pixel in der Höhe angepasst.
             label: Logo
+          previous: Vorherige
+          remove: Entfernen
+          remove_current_banner: Entfernen Sie Die Aktuellen Banner
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Aktuelles Banner entfernen

File: config/locales/hyrax.de.yml [hunk 6]
```diff
@@ -598,15 +602,22 @@ de:
...
             label: Banner
           branding:
             description: Optional können Sie ein Bannerbild und / oder Logo-Bilder hochladen, die mit dieser Sammlung verknüpft werden sollen. Wenn diese hochgeladen werden, werden diese Bilder oben auf der Sammlungsseite angezeigt, um ein eindeutiges Markenbildung für die Sammlung bereitzustellen.
             label: Markenbildung
+          choose_file: Wählen Sie "Datei"
+          link_url: 'Link-URL:'
           logo:
             description: Ein oder mehrere Bilder, die oben auf der Sammlungsseite angezeigt werden. Für beste Ergebnisse laden Sie ein Bild (JPG, GIF oder PNG) hoch, das 40 Pixel hoch ist. Größere Bilder werden auf 40 Pixel in der Höhe angepasst.
             label: Logo
+          previous: Vorherige
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Vorherig

File: config/locales/hyrax.de.yml [hunk 7]
```diff
@@ -598,15 +602,22 @@ de:
...
+          remove: Entfernen
+          remove_current_banner: Entfernen Sie Die Aktuellen Banner
+          remove_logo: Logo Entfernen
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Logo entfernen

File: config/locales/hyrax.de.yml [hunk 8]
```diff
@@ -633,7 +644,13 @@ de:
           add_sharing: Freigaben hinzufügen
           add_user: Benutzer hinzufügen
           current_shared: Derzeit freigegeben für
+          depositor: Einleger
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Deponent

File: config/locales/hyrax.de.yml [hunk 9]
```diff
@@ -633,7 +644,13 @@ de:
...
+          manager: Manager
           note: Unabhängig von den Sichtbarkeitseinstellungen dieser Sammlung können Sie diese Sammlung für bestimmte Gruppen und Benutzer freigeben.
+          search_for_a_group: Suche für eine Gruppe...
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Eine Gruppe suchen...

File: config/locales/hyrax.de.yml [hunk 10]
```diff
@@ -633,7 +644,13 @@ de:
...
+          search_for_a_user: Suche für einen Benutzer...
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Einen Benutzer suchen...

File: config/locales/hyrax.de.yml [hunk 11]
```diff
@@ -633,7 +644,13 @@ de:
...
+          select_a_role: Wählen Sie eine Rolle...
+          viewer: Viewer
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Betrachter

File: config/locales/hyrax.de.yml [hunk 12]
```diff
@@ -674,6 +691,9 @@ de:
           show_less_parent_collections: "...zeige weniger"
           show_more_parent_collections: Zeig mehr...
           subcollection_count: Untersammlungen
+        sort_and_per_page:
+          show_par_page_html: Karte %{select} pro Seite
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Zeige %{select} pro Seite

File: config/locales/hyrax.de.yml [hunk 13]
```diff
@@ -674,6 +691,9 @@ de:
...
+          sort_by: 'Sortieren Nach:'
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: 'Sortieren nach:'

File: config/locales/hyrax.de.yml [hunk 14]
```diff
@@ -797,6 +817,7 @@ de:
         file_views: Aussicht
         files: Dateien deponiert
         heading: Ihre Statistik
+        joined_on: Trat auf
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Beigetreten am

File: config/locales/hyrax.de.yml [hunk 15]
```diff
@@ -897,6 +918,19 @@ de:
         table_title_access: Zugriffsebene
         table_title_user: Person / Gruppe
         user_search: Nach einem Benutzer suchen
+      show_actions:
+        analytics: Analytics
+        confirm_delete_this: Delete %{type}?
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: %{type} löschen?

File: config/locales/hyrax.de.yml [hunk 16]
```diff
@@ -897,6 +918,19 @@ de:
...
+        delete_this: Löschen %{type}
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: %{type} löschen?

File: config/locales/hyrax.de.yml [hunk 17]
```diff
@@ -897,6 +918,19 @@ de:
...
+        edit_this: Bearbeiten %{type}
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: %{type} bearbeiten

File: config/locales/hyrax.de.yml [hunk 18]
```diff
@@ -897,6 +918,19 @@ de:
...
+      show_details:
+        characterization: Charakterisierung
+        date_modified: Datum Geändert
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Letztes Änderungsdatum

File: config/locales/hyrax.de.yml [hunk 19]
```diff
@@ -897,6 +918,19 @@ de:
         table_title_access: Zugriffsebene
         table_title_user: Person / Gruppe
         user_search: Nach einem Benutzer suchen
+      show_actions:
+        analytics: Analytics
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Statistik

File: config/locales/hyrax.de.yml [hunk 20]
```diff
@@ -897,6 +918,19 @@ de:
...
+        confirm_delete_this: Delete %{type}?
+        delete_this: Löschen %{type}
+        edit_this: Bearbeiten %{type}
+      show_details:
+        characterization: Charakterisierung
+        date_modified: Datum Geändert
+        date_uploaded: Datum Hochgeladen
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Hochlade-Datum

File: config/locales/hyrax.de.yml [hunk 21]
```diff
@@ -897,6 +918,19 @@ de:
...
+        depositor: Einleger
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Deponent

File: config/locales/hyrax.de.yml [hunk 22]
```diff
@@ -897,6 +918,19 @@ de:
...
+        file_details: Datei Details
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Dateidetails

File: config/locales/hyrax.de.yml [hunk 23]
```diff
@@ -897,6 +918,19 @@ de:
...
+        fixity_check: Festigkeit Überprüfen
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Beständigkeit überprüfen

File: config/locales/hyrax.de.yml [hunk 24]
```diff
@@ -897,6 +918,19 @@ de:
...
+        not_yet_characterized: Noch Nicht Gekennzeichnet
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Noch nicht charakterisiert

**Comment by jmfrenzel** [comment]
NB: This PR resolves some of missing _i18n listed in #1386

**Comment by cjcolvar** [comment]
@j-dornbusch If you rebase from master, you'll get #3548 which should fix the failing tests due to chromedriver.

**Comment by cjcolvar** [comment]
@j-dornbusch The chromedriver issue should be solved for now so I think it should pass CI if you rebase this PR on master.

**Comment by j-dornbusch** [comment]
@cjcolvar Here, there are no more failures on circleci.

**Comment by cjcolvar** [comment]
@j-dornbusch This is looking good but it looks like your squashed rebase includes the chromedriver commit.  That should already be in master so it should be removed from your commit and the build should still pass.

**Comment by j-dornbusch** [comment]
@cjcolvar Did I squash too many commits ? I tried to cancel the squash but I'm not sure of the result.

**Comment by cjcolvar** [comment]
That took care of the CI build changes that leaked in, but now it looks like something got lost since there are failing tests in `spec/features/collection_multi_membership_spec.rb`.

**Comment by j-dornbusch** [comment]
@cjcolvar I can't reproduce the test failure locally.
```
rspec ./spec/features/collection_multi_membership_spec.rb
...
Finished in 1 minute 1.9 seconds (files took 4 seconds to load)
8 examples, 0 failures
```
I'm on the right branch, everything has been pushed.
`git status`
```
On branch more-i18n-omissions
Your branch is up-to-date with 'origin/more-i18n-omissions'.
```

**Comment by cjcolvar** [comment]
:+1: I restarted the build.  Let's see what happens this time.

**Comment by cjcolvar** [comment]
There are still 3 failures but slightly different than last time.  I'm going to see what I can find out by testing locally.

**Comment by cjcolvar** [comment]
Locally, I'm occasionally seeing one failing test in the `collection_multi_membership_spec` but not the others so I'm rerunning the build again now to see if we can get a passable build so this can be merged.  At that point I'll create an issue for the randomly failing test.

**Comment by cjcolvar** [comment]
Now circle is saying the opposite: only the two service tests are failing:
./spec/services/hyrax/embargo_service_spec.rb:33
./spec/services/hyrax/lease_service_spec.rb:33

@samvera/hyrax-code-reviewers Anyone else have ideas about this?

**Comment by cjcolvar** [comment]
I'm guessing that this is failing for the same reasons as the embargo and lease services are failing in hydra-access-controls: https://travis-ci.org/samvera/hydra-head/jobs/501631455
I'm going to dig into this more now since it could be breaking all new hydra-head dependent apps.

**Comment by cjcolvar** [comment]
@j-dornbusch #3559 got merged.  Do you mind rebasing one more time?

**Comment by j-dornbusch** [comment]
@cjcolvar I rebased and force-pushed. 
```
git checkout master
git fetch upstream
git rebase upstream/master
git checkout more-i18n-omissions
git rebase master
git push --force
```

**Comment by j-dornbusch** [comment]
@cjcolvar I can reproduce the "spec/features/work_show_spec.rb" failure locally, so I will investigate it.

**Comment by j-dornbusch** [comment]
@cjcolvar The test fails randomly locally, but very rarely. By setting breakpoints, I do not notice anything particular. I can not fail it in debug mode, it still succeeds.

**Comment by cjcolvar** [comment]
Somehow that rebase brought in my commits to master from #3559.  I think those should be removed and I'll restart the circle build until it is green.

**Comment by cjcolvar** [comment]
@j-dornbusch Is there anything I can do to help resolve the git history issue and help this get merged?  Would you like me to make a new PR with the rebase reapplied?

**Comment by j-dornbusch** [comment]
Yes @cjcolvar  this PR is now old and we have some tickets on our Hyrax instance that depend on it. If you have a solution I will be grateful.

**Comment by cjcolvar** [comment]
Closing now that #3708 was merged.

**Pull request closed without merging**